src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -425,18 +425,68 @@ public async Task Pound_i_nuget_displays_list_of_added_sources(Language language
 
             using var events = kernel.KernelEvents.ToSubscribedList();
 
-            await kernel.SubmitCodeAsync(@"#i ""nuget:https://completelyFakerestoreSource""");
+            await kernel.SubmitCodeAsync(@"
+#i ""nuget:https://completelyFakerestoreSource1""
+#i ""nuget:https://completelyFakerestoreSource2""
+");
 
-            events.Should()
-                  .ContainSingle<DisplayedValueUpdated>()
-                  .Which
-                  .FormattedValues
-                  .Should()
-                  .ContainSingle(v => v.MimeType == "text/html")
-                  .Which
-                  .Value
-                  .Should()
-                  .ContainAll("Restore sources", "https://completelyFakerestoreSource");
+            Assert.Collection(events,
```

Comment:
Could you use FluentAssertions and make the specification a little more focused? Maybe try `events.Should().ContainInOrder(...)` and only specify the events we care about?

Reply:
@jonsequitur , lordy ... I am not fluent in fluentassertions, but yes, I can try.